compiler/tests/success/x86-64/opsizes.jazz

```diff
@@ -6,9 +6,9 @@ u = v;
 x = x + s;
 x = x + 64;
 x = x + 64s s;
-x = x >> u;
-x = x >>s 64;
-x = x >>64s u;
+x >>= u & 63;
+x = x >>s 24;
```

Comment:
Why `64` -> `24`? And why not write `x >>s= 24`?

Reply:
1. Because 64 is bigger than 63.
2. I kept the syntax that was there before, without the compound assignment.